packages/headless/src/user.ts

```diff
@@ -90,6 +90,50 @@ export function useUpdateUser({
   });
 }
 
+export function useUpdateUserPersonalInfo({
+  onSuccess,
+  onError,
+}: {
+  onSuccess?: OnSuccess<IUser.Self>;
+  onError?: OnError;
+}) {
+  const atlas = useAtlas();
+  const update = useCallback(
+    async ({ id, payload }: { id: number; payload: IUser.UpdateApiPayload }) =>
+      atlas.user.update(id, payload),
+    [atlas.user]
+  );
+
+  return useMutation({
+    mutationFn: update,
+    mutationKey: [MutationKey.UpdateUserPersonalInfo],
+    onSuccess,
+    onError,
+  });
+}
+
+export function useUpdateTutorTopics({
+  onSuccess,
+  onError,
+}: {
+  onSuccess?: OnSuccess<void>;
+  onError?: OnError;
+}) {
+  const atlas = useAtlas();
+  const update = useCallback(
+    async ({ payload }: { payload: ITopic.ReplaceUserTopicsApiPayload }) =>
```

Comment:
No need to add an param-object. 

```suggestion
    async (payload:  payload: ITopic.ReplaceUserTopicsApiPayload) =>
```